lib/web_ui/dev/felt_incremental.bat

```diff
@@ -0,0 +1,70 @@
+:: felt_windows: a command-line utility for Windows for building and testing
```

Comment:
Are these changes supposed to be in this PR? They seem unrelated?

Reply:
Some of the other files as well...

Reply:
Yeah, feel free to ignore. I will split this up into multiple PRs. Just sent this draft hastily as I was rushing out the door for vacation :-)